MixedRemoteViewCompositor/Source/MixedRemoteViewCompositor.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15

```

Comment:
there are additional changes needed for x64, latest SDK and to build correctly without warnings for 2017. See my repo for a version that compiles cleanly:
https://github.com/carmines/MixedRealityCompanionKit/tree/master/MixedRemoteViewCompositor